### PR TITLE
Update for Issue 152 - Include development/testing in Landscape

### PR DESCRIPTION
**Pull request by andrew-est**
Update for https://github.com/cncf/tag-env-sustainability/issues/152

This is an initial attempt to include development/testing in the landscape document at the same level as Other workloads. If the conversations leads to other locations or proposals thats all the better.

### CODE REVIEW

**Comment by leonardpahlke** [comment]
PTAL @catblade

File: website/content/about/SustainabilityUseCasesAndLandscape2023.md [hunk 1]
```diff
@@ -169,6 +169,16 @@ In either case, AI/Ml clusters often  have the added complexity of XPUs, or acce
...
+While a concentration is made on the sustainability of active services used by consumers a major factor in the overall footprint of cloud
+sustainability is in the development and testing of the software that these services use. 
+The environmental footprint of the development of software has the same factors that effect it as actual services running but they may be weighted differently. 
+Examples of the types of loads during software development are the individual hardware costs of the developers, the build servers for the software and the various testing stages in the creation of services. 
+Each of these stages are necessary for the development of software but incur an environmental cost. The different loads can be addressed in different ways with flexibility that are different from the sector specific challenges that the software is being produced for. 
+Examples of this is that the testing of Telecoms software might not be as time critical as the production deployments and this gives different flexibility. While there can be some greater flexibility with the location of build servers there will still be speed constraints and developer efficiency is important.
```
Comment (catblade) [inline website/content/about/SustainabilityUseCasesAndLandscape2023.md:178]:
```suggestion
For example, application testing may not need the same availability as production deployments. In the test environment, the positioning of the servers may be less critical, but there will still be speed developer efficiency that should be taken into consideration. 
```

File: website/content/about/SustainabilityUseCasesAndLandscape2023.md [hunk 2]
```diff
@@ -169,6 +169,16 @@ In either case, AI/Ml clusters often  have the added complexity of XPUs, or acce
...
+The environmental footprint of the development of software has the same factors that effect it as actual services running but they may be weighted differently. 
+Examples of the types of loads during software development are the individual hardware costs of the developers, the build servers for the software and the various testing stages in the creation of services. 
+Each of these stages are necessary for the development of software but incur an environmental cost. The different loads can be addressed in different ways with flexibility that are different from the sector specific challenges that the software is being produced for. 
+Examples of this is that the testing of Telecoms software might not be as time critical as the production deployments and this gives different flexibility. While there can be some greater flexibility with the location of build servers there will still be speed constraints and developer efficiency is important. 
+The location and utilization of test systems which can be long running and large would seem to be an area of a lot of concern as they can sometimes be of the same order of magnitude as the active deployed systems.
```
Comment (catblade) [inline website/content/about/SustainabilityUseCasesAndLandscape2023.md:179]:
```suggestion
These test systems may still be large and long-run, but would seem to be an area of concern regarding sustainability optimization for the full system.
```

File: website/content/about/SustainabilityUseCasesAndLandscape2023.md (hunk 1 above)
Comment (catblade) [reply website/content/about/SustainabilityUseCasesAndLandscape2023.md:177]: Still outstanding, and open to rephrasing.

**Review by guidemetothemoon — APPROVED** [review]
LGTM! 😊

File: website/content/about/SustainabilityUseCasesAndLandscape2023.md (hunk 1 above)
Comment (catblade) [reply website/content/about/SustainabilityUseCasesAndLandscape2023.md]: No update.  Conversation was just marked resolved, or am I missing something?

**Comment by leonardpahlke** [comment]
@catblade where are we on this PR?

**Comment by catblade** [comment]
> @catblade where are we on this PR?

@leonardpahlke  still waiting on updates/responses to our reviews.

**Review by andrew-est — COMMENTED** [review]
I'm happy with the changes and want to get it merged

File: website/content/about/SustainabilityUseCasesAndLandscape2023.md [hunk 3]
```diff
@@ -169,6 +169,15 @@ In either case, AI/Ml clusters often  have the added complexity of XPUs, or acce
 These accelerators take significant amounts of power to run, more by an order of magnitude required from regular computer chips.
 Additionally, some of the workloads on these clusters are not time-sensitive, for instance training sets of information, and some are time-sensitive, for instance inference jobs for recognition systems.
 
+#### Software Development and Testing
+Usually, the primary focus of environmental sustainability is restricted to active services.  However, there is also a substantial footprint that can be attributed to the development and testing of the software these services rely on.
```
Comment (catblade) [inline website/content/about/SustainabilityUseCasesAndLandscape2023.md:173]: Can you define active services here?

File: website/content/about/SustainabilityUseCasesAndLandscape2023.md [hunk 4]
```diff
@@ -169,6 +169,15 @@ In either case, AI/Ml clusters often  have the added complexity of XPUs, or acce
...
+This footprint may be weighted differently than the one of the services.
+Some examples of these loads during software development are the individual hardware costs of the developers, the build servers for the software, and the various testing stages in the creation of services. 
+Each of these stages is necessary for the development of software, but they do incur an environmental cost. These loads should be addressed in a way that makes sense for this area. 
+For example, application testing may not need the same availability as production deployments. In the test environment, the positioning of the servers may be less critical, but there will still be speed developer efficiency that should be taken into consideration. 
+These test systems may still be large and long-run, but would seem to be an area of concern regarding sustainability optimization for the full system.
```
Comment (catblade) [inline website/content/about/SustainabilityUseCasesAndLandscape2023.md:178]:
```suggestion
These test systems may still be large and lengthy and have an impact regarding power and carbon optimization for the full system.
```

**Review by catblade — COMMENTED** [review]
Good start.  Added some commentary to help tune this.

**Comment by SamYuan1990** [comment]
Hi guys,
I hope we can rebase this PR to make it up to date.
for example, considering with i18n comparing https://github.com/cncf/tag-env-sustainability/tree/main/website/content and https://github.com/Nordix/wg-env-sustainability/tree/issue-152/website/content the folder structure is changed. and the branch is over 100 commits behind the latest.
> This branch is [9 commits ahead of](https://github.com/Nordix/wg-env-sustainability/compare/cncf%3Atag-env-sustainability%3Amain...issue-152), [162 commits behind](https://github.com/Nordix/wg-env-sustainability/compare/issue-152...cncf%3Atag-env-sustainability%3Amain) cncf/tag-env-sustainability:main.

**Comment by leonardpahlke** [comment]
I am closing this PR since its now open for a long time and it seems like the proposed changes do not get merged any time soon. Feel free to reopen this PR if you like to get it merged.

**Pull request closed without merging**